packages/valhalla-api/PKGBUILD

```diff
@@ -1,17 +1,17 @@
 # This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
 # See COPYING for license details.
 
-pkgname=valhalla
+pkgname=valhalla-api
```

Comment:
I checked, there is no BA package depending on it, so there is no other package to update.

Reply:
Good. Let's wait for PR merging.